antsibull/data/collection-enum.py

```diff
@@ -124,6 +128,49 @@ def load_all_plugins(plugin_type, basedir, coll_filter):
     return result
 
 
+def load_collection_meta_manifest(b_manifest_path):
+    with open(b_manifest_path, 'rb') as f:
+        meta = json.load(f)
+    return {
+        'namespace': meta['collection_info']['namespace'],
+        'name': meta['collection_info']['name'],
+        'version': meta['collection_info']['version'],
+    }
+
+
+def load_collection_meta_galaxy(b_galaxy_path):
+    with open(b_galaxy_path, 'rb') as f:
+        meta = yaml.safe_load(f)
+    return {
+        'namespace': meta['namespace'],
+        'name': meta['name'],
+        'version': meta.get('version'),
```

Comment:
Did you drop the `'*'` fallback intentionally?

Reply:
`*` was put there by Ansible code (https://github.com/ansible/ansible/blob/devel/lib/ansible/galaxy/collection/__init__.py#L486), not by the collections. (Or at least no collection I know of.)

Reply:
Fair enough. It's a value for requested requirements, not concrete resolved collections.